src/core/components/web-view/index.web.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Dimensions } from 'react-native';
+import React from 'react';
+
+// Currently partially implemented, should be brought to parity
+// with mobile in https://github.com/Instawork/hyperview/issues/455
+export default (props: any) => {
+  const windowDimensions = Dimensions.get('window');
+  const windowHeight = windowDimensions.height;
+  const windowWidth = windowDimensions.width;
+  if (props.source.uri) {
+    return (
+      <iframe
+        height={windowHeight}
+        src={props.source.uri}
+        title="-"
```

Comment:
is a title required?

Reply:
ESLint is complaining when it's missing:
<img width="695" alt="Screen Shot 2022-07-17 at 4 16 31 PM" src="https://user-images.githubusercontent.com/309515/179428759-8426bcc3-0d8a-40c6-a0a7-916204d75693.png">